integration_tests/support/commands.ts

```diff
@@ -23,8 +24,8 @@ const logAccessibilityViolations = (violations: Array<Result>) => {
   cy.task('table', violationsData)
 }
 
-Cypress.Commands.add('checkAccessibility', () => {
+Cypress.Commands.add('checkAccessibility', (rules: AxeRules = undefined) => {
   cy.injectAxe()
   cy.configureAxe({ rules: [{ enabled: false, id: 'region', selector: '.govuk-phase-banner' }] })
-  cy.checkA11y(undefined, undefined, logAccessibilityViolations)
+  cy.checkA11y(undefined, { rules }, logAccessibilityViolations)
```

Comment:
Are we always passing `{ rules: undefined | AxeRules }` here? Is it ok to be passing `{ rules: undefined }` if so? Not saying it's a problem, just curious - if it works, it works!

Reply:
Yeah there doesn't appear to be any issues with it being undefined.  I'm confident it works as I've got it on another branch with the integration tests for the Select programme page.